CS342MemberViews/Sources/SonyaJin/ProfileView.swift

```diff
@@ -0,0 +1,37 @@
+//
+// This source file is part of the CS342 2023 Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+import SwiftUI
+
+
+public struct ProfileView: View {
+    public var body: some View {
+        profileImage
+            .resizable()
+            .aspectRatio(1.0, contentMode: .fit)
+            .clipShape(Circle())
+            .shadow(radius: 10)
+            .padding(.horizontal, 32)
+            .frame(maxWidth: 300)
+    }
+    
+    private var profileImage: Image {
+        guard let imagePath = Bundle.module.path(forResource: "SonyaJin", ofType: "jpeg"),
```

Comment:
Your image is not displayed in the view. This is due to the fact that you have added the wrong path extension:
```suggestion
        guard let imagePath = Bundle.module.path(forResource: "SonyaJin", ofType: "jpg"),
```

Reply:
@sonyashijin I have committed a fix to your issue here to get you started with adding some more creative views.